com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/MyTextDocumentService.java

```diff
@@ -166,10 +167,10 @@
   @Override
   public void didOpen(DidOpenTextDocumentParams params) {
     String uri = params.getTextDocument().getUri();
-    // A better implementation that will cover the gitfs scenario will be implementated later based
-    // on issue #173
+
+    // git FS URIs are not currently supported
     if (uri.startsWith(GIT_FS_URI)) {
-      communications.notifyThatExtensionIsUnsupported("gitfs");
+      log.warn(String.join(" ", GITFS_URI_NOT_SUPPORTED, uri));
```

Comment:
So you are going to log it only on the server, not on the client?

Reply:
Yes because is an information that give no value to the user since with git filesystem we still analyze the document so the message is pretty useless, but I could implement the things in a different way if needed

Reply:
BTW if user open a copybook file we still throw a popup message to the user

Reply:
Ok, I got it, it's fine

Reply:
;)